src/server/controllers/auth.js

```diff
@@ -3,18 +3,19 @@ const router = express.Router();
 
 const connection = require('../airtable');
 const mapColumns = require('../utils/mapColumns');
+const user = require('../utils/user');
```

Comment:
Rename this to something a little more obvious like `userUtility` or somesuch

Reply:
That thought did occur to me, but no good name sprang to mind. I'll borrow yours for now, but I'm open to suggestions otherwise :P